scripts/coverage.sh

```diff
@@ -22,4 +20,7 @@ if [ ! $trpc_running ]; then
   trpc_pid=$!
 fi
 SOLIDITY_COVERAGE=true && ./node_modules/.bin/solidity-coverage
```

Comment:
Yes that's correct - it prevents HDWallet from being instantiated in `truffle.js` when coverage runs. Should just read as below?
```
SOLIDITY_COVERAGE=true  ./node_modules/.bin/solidity-coverage
```